git-hooks.gradle.kts

```diff
@@ -0,0 +1,24 @@
+tasks.register("copyGitHooks", Copy::class.java) {
+    description = "Copies the git hooks from /git-hooks to the .git folder."
+    group = "git hooks"
+    from("$rootDir/hooks/pre-commit")
+    into("$rootDir/.git/hooks/")
+}
+
+tasks.register("installGitHooks", Exec::class.java) {
+    description = "Installs the pre-commit git hooks from /git-hooks."
+    group = "git hooks"
+    workingDir = rootDir
+    commandLine = listOf("chmod")
+    args("-R", "+x", ".git/hooks/")
+    dependsOn("copyGitHooks")
+    doLast {
+        logger.info("Git hook installed successfully.")
+    }
+}
+
+afterEvaluate {
+    tasks.findByName("clean")?.dependsOn(":installGitHooks")
+    tasks.findByName("assemble")?.dependsOn(":installGitHooks")
+    tasks.getByPath(":app-android:preBuild").dependsOn(":installGitHooks")
+}
```

Comment:
You can change the git hooks directory from `.git/hooks` to any directory you want. It's better to use the custom hook directory rather than copying hook scripts frequently.

Please add these lines to /.gitignore.

```
# Ignore files under /hooks except pre-defined hooks
/hooks/*
!/hooks/pre-commit
```

and then, create a task in the */build.gradle.kts*.

```suggestion
tasks.register("installGitHooks", Exec::class.java) {
    description = "Set up local git config to use /hooks for git-hooks"
    group = "git hooks"
    workingDir = layout.projectDirectory.getAsFile()
    commandLine = listOf("git")
    args("config", "--local", "core.hooksPath", "hooks")
}
```

Lastly, please make reference to `./gradlew installGitHooks` in [README.md#Requirements](https://github.com/DroidKaigi/conference-app-2023#requirements).